openfecli/commands/gather.py

```diff
@@ -524,6 +532,26 @@ def _get_legs_from_result_jsons(
             legs[names][simtype].extend(dGs)
     return legs
 
+
+def rich_print_to_stdout(df: pd.DataFrame) -> None:
+    """Use rich to pretty print a table to stdout."""
+
+    from rich.console import Console
+    from rich.table import Table
+    from rich import box
+
+    table = Table(box=box.MINIMAL_HEAVY_HEAD)
+    for col in df.columns:
+        table.add_column(col)
+
+    for value_list in df.values.tolist():
+        row = [str(x) for x in value_list]
+        table.add_row(*row)
```

Comment:
This is all super clear, but does rich have any built in functions to work with a pandas df?

Reply:
surprisingly, no!